src/lib/useIsEclipticOwner.js

```diff
@@ -0,0 +1,20 @@
+import { useWallet } from 'store/wallet';
+import { usePointCache } from 'store/pointCache';
+
+import { eqAddr } from 'lib/wallet';
+
+export default function useIsEclipticOwner() {
```

Comment:
Do we really need a helper thing for this? It's only ever gonna be used in one place...

Reply:
I guess not, but it also feels cleaner when it's nice and isolated like this